StripeUICore/StripeUICore/Source/Elements/Section/SectionContainerView.swift

```diff
@@ -31,25 +31,27 @@ class SectionContainerView: UIView {
     }()
 
     lazy var stackView: StackViewWithSeparator = {
-        let view = buildStackView(views: views, theme: theme)
+        let view = buildStackView(views: views, theme: theme, setMask: setMask)
```

Comment:
I think this is getting closer to what we want, but I think `setMask` is a little too vague, and I don't think it should be a required parameter for creating a SectionContainerView. We should look at doing something like adding a function/API to the SectionContainerView to put it in a "disabled" state.

Reply:
I've added a function to the SectionElement called disableAppearance, and it calls a function on the SectionContainerView that applies the mask